addon/components/nrg-application.hbs

```diff
@@ -5,7 +5,9 @@
     {{if this.computerScreenSidebarActive "large-screen-sidebar-active"}}"
   {{on-resize this.setMainContentStyle}}
 >
-  <NrgAppbar @title={{this.title}} @onToggleSidebar={{this.toggleSidebar}} />
```

Comment:
If you're removing the title attribute, does that property need to be removed from the component? Or does it need to be removed at all?

Reply:
We probably want to remove it altogether. I'll look for somewhere we are using it.